app/components/Onboarding/BtcPayServer.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './BtcPayServer.scss'
+
+const BtcPayServer = ({
+  connectionString,
+  connectionStringIsValid,
+  setConnectionString,
+  startLndHostError
+}) => (
+  <div className={styles.container}>
+    <section className={styles.input}>
+      <label htmlFor="connectionString">Connection String:</label>
+      <textarea
+        type="text"
+        id="connectionString"
+        rows="9"
+        placeholder="BTCPayServer Connection String"
+        className={
+          connectionString && (startLndHostError || !connectionStringIsValid)
+            ? styles.error
+            : undefined
+        }
+        ref={input => input}
+        value={connectionString}
+        onChange={event => setConnectionString(event.target.value)}
+      />
+      <p
+        className={`${styles.errorMessage} ${
+          connectionString && !connectionStringIsValid ? styles.visible : undefined
```

Comment:
nit: explicit precedence

Reply:
Can you explain what you mean by this please @Empact ? Thanks.

Reply:
I prefer making precedence explicit anywhere there is potential ambiguity, in this case between `&&` & `?`. e.g.

`(connectionString && !connectionStringIsValid) ? styles.visible : undefined`

Reply:
Our lint rules do not allow for this. Prettier automatically removes the brackets.